references/data-converters-shared-flow/sharedflowbundle/resources/jsc/convert-odata.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var input = context.getVariable("dataconverter.input");
+var resource = context.getVariable("dataconverter.resource");
+
+var newBody = ConvertODataToRest(JSON.parse(input), resource);
```

Comment:
can we rename this function to `convertODataResponse` or something along these lines to make it more consistent with the rest?

Reply:
yep, will do